Source/GmmLib/CMakeLists.txt

```diff
@@ -545,3 +545,4 @@ if(ARCH EQUAL 32)
 endif()
 
 add_subdirectory(ULT)
+install(TARGETS gmm_umd ARCHIVE DESTINATION lib)
```

Comment:
Unfortunately that's not enough. It is important to install header files as well for other projects like media-driver to be able to link against gmmlib.

Reply:
@dvrogozh are all headers in "Source/inc"? Should the same dir structure (common, umKmInc) be installed to the system include dir?

Reply:
I afraid they are scattered:(. This is highlighted here: https://github.com/intel/gmmlib/issues/5. The problem is that I don't even know which include files are needed and which are not (sorry I am not in the gmmlib team). Probably the best way to approach this would be to take some project which depends on gmmlib, for example media-driver, wipe out current detection of gmmlib and substitute it with the proper detection by gmmlib.pc, after that start add include files one by one depending on compilation errors.